devfiles/osio-dogfooding.devfile.yaml

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
```

Comment:
@vitaliy-guliy looks good, just a note that `osio` is sorta deprecated name consider using `hosted-che` instead